testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -130,6 +150,44 @@ class TestOpenMMTopologyParser(OpenMMTopologyBase):
     expected_n_bonds = 1922
 
 
+class TestOpenMMTopologyParserWithNoElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    def test_with_partial_elements(self, top):
+        if 'elements' in self.expected_attrs:
+            omm_topology = self.ref_filename
+            wmsg = ("Unknown element None found for some atoms. "
+                    "These have been given an empty element record "
+                    "with their atomtype set to 'X' "
+                    "and their mass set to 0.0. "
+                    "If needed they can be guessed using "
+                    "MDAnalysis.topology.guessers.")
+            with pytest.warns(UserWarning, match=wmsg):
+                mda_top = self.parser(PDB) \
+                          ._mda_topology_from_omm_topology(omm_topology)
+
+    def test_with_no_elements(self, top):
+        self.expected_attrs.remove('elements')
```

Comment:
As `expected_attrs` is a class attribute, I think this will remove "elements" from the base class and cause potential trouble for any further subclasses of OpenMMTopologyBase. Could you please just create re-define `expected_attrs`? I think that also means that another check will run that the resulting Universe does *not* have elements.